lib/dataset-tools/augmentation/replace_parameters.ts

```diff
@@ -751,7 +754,10 @@ export default class ParameterReplacer {
             }
 
             if (!type.isNumeric()) {
-                if (this._paramLangPack.isGoodPersonName(sampled))
+                if (
+                    (this._paramLangPack.isGoodPersonName(sampled)) ||
+                    (this._paramLangPack.isGoodUserName(sampled) && typeValue && typeValue === "tt:username")
+                )
```

Comment:
By the way, this indentation style is weird. I would write as:
```
if ((this._paramLangPack.isGoodPersonName(sampled)) ||
    (this._paramLangPack.isGoodUserName(sampled) && typeValue && typeValue === "tt:username")) {
```
It feels more "normal" to me, in a C-style language.